packages/grafana-ui/src/components/Forms/Form.tsx

```diff
@@ -1,24 +1,27 @@
 import React, { useEffect } from 'react';
 import { useForm, Mode, OnSubmit, DeepPartial, FormContextValues } from 'react-hook-form';
 
-type FormAPI<T> = Pick<FormContextValues<T>, 'register' | 'errors' | 'control'>;
+type FormAPI<T> = Pick<FormContextValues<T>, 'register' | 'errors' | 'control' | 'getValues'>;
```

Comment:
@dprokop can I export this so I can reuse it in ImportDashboardForm?

Reply:
yes, I think you need to.